Tatoeba/tatoeba2
# Rename Toki Pona code

Solves #2898.

Deployment instructions (as root):

1. Disable reindexation in crontab (both merge and delta update)
    ```
    crontab -e
    ```
2. Make sure latest indexation finished
    ```
    [ -f /tmp/SphinxIndexesShell.php.lock ] && echo "Wait" || echo "Good to go"
    ```
3. Turn on maintenance mode
    ```
    vim /etc/nginx/sites-enabled/tatoeba.org.conf
    systemctl reload nginx
    ```
4. [Deploy](https://github.com/Tatoeba/tatoeba2/wiki/Deployment) (`set-prod-folder.sh` should run the migration script)
5. Generate new Toki Pona indexes
    ```
    cake sphinx_indexes update main tok
    cake sphinx_indexes update delta tok
    ```
6. Turn off maintenance mode
    ```
    vim /etc/nginx/sites-enabled/tatoeba.org.conf
    systemctl reload nginx
    ```
7. Enable reindexation in crontab (both merge and delta update)
    ```
    crontab -e
    ```
8. Cleanup old Toki Pona index files
    ```
    rm /var/manticore/indices/toki_*
    ```
